sdk/storage/azblob/internal/shared/mmf_windows.go

```diff
@@ -14,11 +14,11 @@ import (
 	"unsafe"
 )
 
-// mmb is a memory mapped buffer
-type mmb []byte
+// Mmb is a memory mapped buffer
+type Mmb []byte
 
-// newMMB creates a new memory mapped buffer with the specified size
-func newMMB(size int64) (mmb, error) {
```

Comment:
do we want to expose these to the customer?

Reply:
These were un-exported always. We can discuss this, and take it in a new review if we decide to change..

Reply:
I think this is an internal implementation and we should not expose this to the customer.